### PR TITLE
Alpha themelistbox

### PR DESCRIPTION
**Pull request by Wagnerp**
* Use file name as theme name

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 namespace Krypton.Toolkit
 {
-    public partial class KryptonThemeBrowserForm : KryptonForm
+    internal partial class KryptonThemeBrowserForm : KryptonForm
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs:11]: This stops it from being used by external Users of the Suite. Is that the intention ?

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs [hunk 2]
```diff
@@ -38,7 +38,7 @@ public KryptonThemeBrowserForm(FormStartPosition? startPosition = FormStartPosit
 
             _formStartPosition = startPosition ?? FormStartPosition.CenterScreen;
 
-            _startIndex = startIndex ?? ThemeManager.GetThemeIndex();
+            _startIndex = startIndex ?? 0; // ThemeManager.GetThemeIndex();
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs:41]: Why is `startIndex` a nullable item, just make it zero

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs:11]: I was going to make a public API for it via `KryptonThemeBrowser`, but the theme index numbers seem to be broken

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs:41]: Need to figure out why the theme index numbers are broken

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs]: Reverted changes

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs]: Reverted changes

**Pull request merged**